config/environments/development.rb

```diff
@@ -50,6 +50,12 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :letter_opener
```

Comment:
Kindly fix this issue, When I tried to reset the passsword I got this error : 
![image](https://github.com/Ogaga01/Blog-App/assets/87448628/e689ed95-3a9d-4c78-9278-dab368a9c9f7)

Tip:  Try fixing this error by removing this line(53) from development.rb file after you add confirmable to devise.